sdk/storage/storage-blob/test/blobclientpollers.spec.ts

```diff
@@ -143,7 +146,8 @@ describe("BlobClient beginCopyFromURL Poller", () => {
     const poller2 = await newBlobClient.beginCopyFromURL(
       "https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/README.md",
       {
-        resumeFrom: state
+        resumeFrom: state,
+        ...testPollerProperties
```

Comment:
Nit: You'd typically use the spread __before__ your other options, since if a field exists in the object you're performing the spread operator on, it'll overwrite the one defined earlier.

e.g.
```js
{
  ...testPollerProperties,
  resumeFrom: state
}
```
This won't cause any problems today, but could if additional properties were added to `testPollerProperties` in the future.

Otherwise, awesome fix!

Reply:
Thanks! I do understand this. Let’s not generate another build just for this reason.